packages/ui/src/components/Badge.tsx

```
@@ -1 +1 @@
-export { Badge } from "@hebo/ui/_shadcn/ui/badge";
+export { Badge } from "~/_shadcn/ui/badge";
```

ESLint

[error] Resolve error: EACCES: permission denied, open '/ZhbwjEUHtt'
    at Object.writeFileSync (node:fs:2409:20)
    at l (/home/jailuser/git/node_modules/.pnpm/get-tsconfig@4.10.1/node_modules/get-tsconfig/dist/index.cjs:7:13685)
    at createFilesMatcher (/home/jailuser/git/node_modules/.pnpm/get-tsconfig@4.10.1/node_modules/get-tsconfig/dist/index.cjs:7:14437)
    at Object.resolve (/home/jailuser/git/node_modules/.pnpm/eslint-import-resolver-typescript@4.4.4_eslint-plugin-import@2.32.0_eslint@9.34.0_jiti@2.5.1_/node_modules/eslint-import-resolver-typescript/lib/index.cjs:298:107)
    at withResolver (/home/jailuser/git/node_modules/.pnpm/eslint-module-utils@2.12.1_@typescript-eslint+parser@8.41.0_eslint@9.34.0_jiti@2.5.1__t_d6dc2b664be10a7fa9f69f99bc417359/node_modules/eslint-module-utils/resolve.js:180:23)
    at fullResolve (/home/jailuser/git/node_modules/.pnpm/eslint-module-utils@2.12.1_@typescript-eslint+parser@8.41.0_eslint@9.34.0_jiti@2.5.1__t_d6dc2b664be10a7fa9f69f99bc417359/node_modules/eslint-module-utils/resolve.js:201:22)
    at relative (/home/jailuser/git/node_modules/.pnpm/eslint-module-utils@2.12.1_@typescript-eslint+parser@8.41.0_eslint@9.34.0_jiti@2.5.1__t_d6dc2b664be10a7fa9f69f99bc417359/node_modules/eslint-module-utils/resolve.js:217:10)
    at resolve (/home/jailuser/git/node_modules/.pnpm/eslint-module-utils@2.12.1_@typescript-eslint+parser@8.41.0_eslint@9.34.0_jiti@2.5.1__t_d6dc2b664be10a7fa9f69f99bc417359/node_modules/eslint-module-utils/resolve.js:233:12)
    at ExportMapBuilder.get (/home/jailuser/git/node_modules/.pnpm/eslint-plugin-import@2.32.0_@typescript-eslint+parser@8.41.0_eslint@9.34.0_jiti@2.5.1___a6f5ffeabaa2b70f581cef0687d8cf57/node_modules/eslint-plugin-import/lib/exportMap/builder.js:37:45)
    at checkSpecifiers (/home/jailuser/git/node_modules/.pnpm/eslint-plugin-import@2.32.0_@typescript-eslint+parser@8.41.0_eslint@9.34.0_jiti@2.5.1___a6f5ffeabaa2b70f581cef0687d8cf57/node_modules/eslint-plugin-import/lib/rules/named.js:46:44)

(import/named)